src/stories/index.tsx

```diff
@@ -3,5 +3,7 @@ import { storiesOf } from "@storybook/react";
 import Example from "../components/Example";
 import "../style/index.css";
 import "../style/fonts.css";
+import Footer from "../components/footer";
 
 storiesOf("Example", module).add("example", () => <Example />);
+storiesOf("Footer", module).add("responsive footer", () => <Footer />);
```

Comment:
I think "Footer" is enough for name, so remove "responsive footer"

Reply:
and add stories for icons

Reply:
done